types/uuid/index.d.mts

```diff
@@ -0,0 +1,10 @@
+import uuid from './index.js';
```

Comment:
This is pretty much a clone of the [wrapper](https://github.com/uuidjs/uuid/blob/main/wrapper.mjs) in uuid itself (though using ts `import` instead of `const` to ensure all symbol meanings are reexported).